spot_driver_cpp/include/spot_driver_cpp/images/images_middleware_handle.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 Boston Dynamics AI Institute LLC. All rights reserved.
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <rclcpp/node.hpp>
+#include <set>
+#include <spot_driver_cpp/images/spot_image_publisher.hpp>
+#include <spot_driver_cpp/interfaces/rclcpp_logger_interface.hpp>
+#include <spot_driver_cpp/interfaces/rclcpp_parameter_interface.hpp>
+#include <spot_driver_cpp/interfaces/rclcpp_tf_interface.hpp>
+#include <spot_driver_cpp/interfaces/rclcpp_wall_timer_interface.hpp>
+#include <string>
+#include <tl_expected/expected.hpp>
+#include <unordered_map>
+
+namespace spot_ros2::images {
+
+struct ImagesMiddlewareHandle : public SpotImagePublisher::MiddlewareHandle {
```

Comment:
Since ImageMiddlewareHandle has private members, I think it's better to make it a `class` instead of a `struct`. This applies to the base class too.

```suggestion
class ImagesMiddlewareHandle : public SpotImagePublisher::MiddlewareHandle {
public:
```